Source/WebCore/platform/graphics/cv/VideoFrameCV.mm

```diff
@@ -130,19 +155,8 @@
     auto source = span;
     auto destination = CVPixelBufferGetSpanOfPlane(rawPixelBuffer, 0);
     size_t bytesPerRowDestination = CVPixelBufferGetBytesPerRowOfPlane(rawPixelBuffer, 0);
-    for (unsigned i = 0; i < height; ++i) {
-        size_t j = 0;
-        while (j < std::min(plane.sourceWidthBytes, bytesPerRowDestination)) {
```

Comment:
This old code ignores `width` entirely when deciding how many bytes to process. Here’s hoping that it’s OK that our newer code does in fact process a number of bytes that’s based on the width.